.github/workflows/frontend.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and Deploy
```

Comment:
Zullen we de commits squashen?  Om het beeld van commits en merges duidelijker te houden.  Zijn er nu 13 i.p.v 1

Reply:
squash wordt in github gedaan